lms/celery.py

```diff
@@ -23,15 +22,16 @@
 APP.config_from_object('django.conf:settings')
 APP.autodiscover_tasks()
 
+# Import after autodiscovery has had a chance to connect to the import_module signal
+# so celery doesn't miss any apps getting installed.
+from django.conf import settings  # pylint: disable=wrong-import-position,wrong-import-order
```

Comment:
@feanil I needed settings here so I imported it similar to how you did in cms/celery.py, is it fine. You can also take a look at this PR as we are planning to merge it today.